.idea/.idea.PetroglyphTools/.idea/modules.xml

```diff
@@ -2,7 +2,7 @@
 <project version="4">
```

Comment:
All the IntelliJ related stuff needs to be cleaned out and the `.gitignore ` updated properly, but that's for a different issue.

Reply:
See #102